Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -884,14 +900,17 @@ class VertexFinder(Enum):
     Iterative = (3,)
 
 
+@acts.examples.NamedTypeArgs(trackSelectorRanges=TrackSelectorRanges)
 def addVertexFitting(
     s,
     field,
     outputDirRoot: Optional[Union[Path, str]] = None,
     associatedParticles: str = "particles_input",
     trajectories: Optional[str] = None,
-    trackParameters: str = "trackparameters",
+    trackParameters: str = "fittedTrackParameters",
+    trackIndices: str = "outputTrackIndices",
     vertexFinder: VertexFinder = VertexFinder.Truth,
+    trackSelectorRanges: Optional[TrackSelectorRanges] = TrackSelectorRanges(),
```

Comment:
```suggestion
    trackSelectorRanges: TrackSelectorRanges = TrackSelectorRanges(),
```
I think it is only `Optional` if the default is `None`.